esphome/onju-voice-microwakeword.yaml

```diff
@@ -241,6 +250,9 @@ esp32_touch:
   denoise_cap_level: L0
 
 binary_sensor:
+  - platform: status
```

Comment:
And what value does that sensor have when the device is not connected? 'Cause I'm guessing it's `unavailable` instead of `off`.

In which case, why not check (from HA) if any exposed entity is `unavailable` instead of adding a new sensor? If you're checking from the ESPHome side, I don't understand where and there are other built-in sensors you can use.

Reply:
Good point, the status sensor doesn't add anything to checking an existing sensor, saves bloating with extra sensors.
Removed and I've reworked my automations to look at switch.user_wake_word state, it's just as fast to change state as the status when you pull/apply the power.